packages/database/src/core/RepoManager.ts

```diff
@@ -55,171 +53,149 @@ export type FirebaseAppLike = FirebaseApp | FirebaseAppExp;
 /**
  * Creates and caches Repo instances.
  */
-export class RepoManager {
-  private repos_: {
-    [appName: string]: {
-      [dbUrl: string]: Repo;
-    };
-  } = {};
-
-  /**
-   * If true, new Repos will be created to use ReadonlyRestClient (for testing purposes).
-   */
-  private useRestClient_: boolean = false;
-
-  static getInstance(): RepoManager {
-    if (!_staticInstance) {
-      _staticInstance = new RepoManager();
-    }
-    return _staticInstance;
-  }
+const repos: {
+  [appName: string]: {
+    [dbUrl: string]: Repo;
+  };
+} = {};
 
-  // TODO(koss): Remove these functions unless used in tests?
-  interrupt() {
-    for (const appName of Object.keys(this.repos_)) {
-      for (const dbUrl of Object.keys(this.repos_[appName])) {
-        repoInterrupt(this.repos_[appName][dbUrl]);
-      }
-    }
-  }
+/**
+ * If true, new Repos will be created to use ReadonlyRestClient (for testing purposes).
+ */
+let useRestClient = false;
 
-  resume() {
-    for (const appName of Object.keys(this.repos_)) {
-      for (const dbUrl of Object.keys(this.repos_[appName])) {
-        repoResume(this.repos_[appName][dbUrl]);
-      }
-    }
+/**
+ * Update an existing repo in place to point to a new host/port.
+ */
+export function repoManagerApplyEmulatorSettings(
```

Comment:
Since repoManager is removed, do we still want the prefix?

Reply:
Yeah, I want to keep it to be able to translate these API calls between iOS, Android and Web.